pyiron_contrib/workflow/channels.py

```diff
@@ -87,24 +87,31 @@ def connect(self, *others: Channel) -> None:
         """
         pass
 
-    def disconnect(self, *others: Channel) -> None:
+    def disconnect(self, *others: Channel) -> list[tuple[Channel, Channel]]:
         """
         If currently connected to any others, removes this and the other from eachothers
         respective connections lists.
 
         Args:
             *others (Channel): The other channels to disconnect from.
+
+        Returns:
+            [list[tuple[Channel, Channel]]]: A list of the pairs of channels that no
+                longer participate in a connection.
         """
+        destroyed_connections = []
         for other in others:
             if other in self.connections:
```

Comment:
Would it maybe deserve a warning if it's not found? This kind of stuff helps debugging

Reply:
Ooops, I forgot to switch branches and added this to the next PR in the stack. Good idea and done!